db/migrate/20201009201842_add_payment_association_to_billing_contract.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentAssociationToBillingContract < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :payments, :billing_contracts, null: false, foreign_key: true
```

Comment:
the same issue, you already have added billing_contract_id into the payments table